### DISCUSSION TITLE
no matching client->server HMAC

### DISCUSSION THREAD

**Discussion by northwest-dev** [spatie/ray#612] [Q&A]
When I try to connect to the Liquid Web server I get:
`Error: Error invoking remote method 'connectSsh': Error: Handshake failed: no matching client->server HMAC`

 I've contacted their support and the SSH connection is working and ok, but they say the app has a cipher issue. 
Their response: 
"I have checked and confirmed that sshd service is working fine in the server as well as able to SSH into server without any issue. Based on the error you have given. I infer the tool you are using not supporting the cipher mentioned.".
"There are bunch of keyalgorithms out there so you would need to check and provide the exact key algorithm the client is using so that we can add that in the sshd configuration to check that resolve the issue or not."

**Comment by AdrianMrn** [comment]
@northwest-dev Could you ask your provider if they support SSH tunneling?

You can also check yourself by following the steps outlined here: https://github.com/spatie/ray/issues/482#issuecomment-880568410
